index.html

```diff
@@ -58,155 +74,175 @@ <h2 class="sub-title">I am a Softwaer Developer</h2>
             </ul>
           
         
-    </section>
-
-
+    </section> 
+    
   <!--Work section -->
-  <section class="grid">
-    <div class="grid-items">
-        <div class="card">
+  <section class="work">
+    <div class="work-items">
+        <!-- <div class="card"> -->
             <div class="card-content">
-                <div class="card-header"><h2>My Recent Works</h2></div>
-                <div class="card-img"></div>
-                <div class="card-main-img"></div>
-                <div class="card-subtitle"><h2>Multi-Post Stories</h2></div>
-                <div class="card-text"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
-                <div class="work-card-buttons">
-                    <ul>
-                        <li><a href="#">css</a></li>
-                        <li><a href="#">html</a></li>
-                        <li><a href="#">javascript</a></li>
-                        <li><a href="#">Ruby</a></li>
-                        
-                    </ul>
+                <div class="work-frist-row">
+                    <div class="card-header"><h2>My Recent Works</h2></div>
+
+                    <div class="card-img"></div>
+                </div>
+                <div class="work-second-row">
+                    <div class="card-main-img"></div>
+                    <article class="work-text">
+                     <div class="card-subtitle"><h2>Multi-Post Stories</h2></div>
+                     <div class="card-text"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
+                     <div class="work-card-buttons">
+                         <ul class="work-button-group">
+                             <li><a href="#">css</a></li>
+                             <li><a href="#">html</a></li>
+                             <li><a href="#">bootstrap</a></li>
+                             <li><a href="#">Ruby</a></li>
+                             
+                         </ul>
+                     </div>
+                     
+                    <button type="button" class="card-about-button">
+                     See Project
+                    </button>
+                    </article>
```

Comment:
- [ ] For the works section, there are 3 columns of cards per row in the template, kindly try to match it 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/215283326-b6c03a72-2a90-44bf-aa85-e7e51c6fdb10.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/215283329-e70665a5-8f88-4b33-9027-21f3ea757d76.png)


- [ ] In the template, each card has 2 states, the default state and the hover state 👇 The card should look like the 1st picture and on hover it should look like the second picture 👍  Kindly try to match the template 💯 

## Default state (Figma):
![image](https://user-images.githubusercontent.com/50721479/215283374-0d4e4adc-3405-4872-b9a6-bf2158a952a8.png)

## Hover state (Figma):
![image](https://user-images.githubusercontent.com/50721479/215283368-03f19cb3-39c6-4199-91f0-6caf9b820c2c.png)